app/src/main/java/com/dcrandroid/util/Utils.java

```diff
@@ -349,8 +349,8 @@ public static void copyToClipboard(Context ctx, String copyText, String successM
         TextView tv = vi.findViewById(android.R.id.message);
         tv.setText(successMessage);
         Toast toast = new Toast(ctx);
-        toast.setGravity(Gravity.TOP | Gravity.CENTER, 0, 50);
-        toast.setDuration(Toast.LENGTH_SHORT);
+        toast.setGravity(Gravity.BOTTOM | Gravity.CENTER, 0, 50);
+        toast.setDuration(Toast.LENGTH_LONG);
```

Comment:
I prefer that the caller should decided the gravity and duration of the Toast. It not necessary to stay long in all cases especially when the Toast text is short.